lib/Field/index.js

```diff
@@ -8,12 +8,13 @@ import FieldValidations from './FieldValidations';
 import FieldActions from './FieldActions';
 import ExpandingTextArea from '../ExpandingTextArea';
 import EditableTextWrapper from '../EditableTextWrapper';
+import { RepeatableSettings } from './RepeatableSettings';
 
 const Field = props => {
-  const classNames = cx(`field field--${props.dir}`, {
+  const classNames = cx(`group field field--${props.dir}`, {
     field__editor: props.canEdit,
     'has-formatting': props.hasFormatting,
-    'is-active': props.isActive,
+    'field-is-active': props.isActive,
```

Comment:
Had to rename this, as we use `.is-active` class all over the place for passing fills down on SVGs